lib/control-app.js

```diff
@@ -74,10 +74,6 @@ module.exports = function (config, cb) {
   // airtap files
   app.use('/__zuul', express.static(__dirname + '/../frameworks'))
 
-  // framework files
-  var frameworkDir = path.resolve(__dirname, '../frameworks/tape')
-  app.use('/__zuul', express.static(frameworkDir))
```

Comment:
This `app.use()` is no longer needed since we no longer serve `tape/framework.js`